Description: Debianize cgi.cfg.
Author: Alexander Wirt <formorer@debian.org>
Forwarded: not-needed

--- a/sample-config/cgi.cfg.in
+++ b/sample-config/cgi.cfg.in
@@ -54,7 +54,22 @@ physical_html_path=@datadir@
 
 url_html_path=@htmurl@
 
+# NAGIOS PROCESS CHECK COMMAND
+# This is the full path and filename of the program used to check
+# the status of the Nagios process.  It is used only by the CGIs
+# and is completely optional.  However, if you don't use it, you'll
+# see warning messages in the CGIs about the Nagios process
+# not running and you won't be able to execute any commands from
+# the web interface.  The program should follow the same rules
+# as plugins; the return codes are the same as for the plugins,
+# it should have timeout protection, it should output something
+# to STDIO, etc.
+#
+# Note: The command line for the check_nagios plugin below may
+# have to be tweaked a bit, as different versions of the plugin
+# use different command line arguments/syntaxes.
 
+icinga_check_command=/usr/lib/nagios/plugins/check_nagios /var/lib/icinga/status.dat 5 '/usr/sbin/icinga'
 
 # URL CGI PATH
 # This is the path portion of the URL that corresponds to the
