Description: Debianize icinga.cfg.
Author: Alexander Wirt <formorer@debian.org>
Forwarded: not-needed

--- a/sample-config/icinga.cfg.in
+++ b/sample-config/icinga.cfg.in
@@ -14,9 +14,14 @@
 # for historical purposes.  This should be the first option specified
 # in the config file!!!
 
-log_file=@LOGDIR@/icinga.log
+log_file=/var/log/icinga/icinga.log
 
+# Commands definitions
+cfg_file=/etc/icinga/commands.cfg
 
+# Debian also defaults to using the check commands defined by the debian
+# nagios-plugins package
+cfg_dir=/etc/nagios-plugins/config
 
 # OBJECT CONFIGURATION FILE(S)
 # These are the object configuration files in which you define hosts,
@@ -24,38 +29,10 @@ log_file=@LOGDIR@/icinga.log
 # Hint: Check the docs/wiki on how to monitor remote hosts with different
 # transport methods and plugins
 
-# You can tell Icinga to process all config files (with a .cfg
-# extension) in a particular directory recursively by using the
-# cfg_dir directive as shown below with conf.d
-
-cfg_dir=@sysconfdir@/conf.d
-
-# or you can split your object definitions across several config files
-# and include them one by one - see the provided sample config
-# in objects/
-#
-# ATTENTION: do not mix cfg_file and cfg_dir such as
-# cfg_file=foo/bar.cfg
-# cfg_dir=foo
-# this will include bar.cfg 2 times and lead to config verification errors.
-
-# You can specify individual object config files as shown below:
-cfg_file=@sysconfdir@/objects/commands.cfg
-cfg_file=@sysconfdir@/objects/contacts.cfg
-cfg_file=@sysconfdir@/objects/timeperiods.cfg
-cfg_file=@sysconfdir@/objects/templates.cfg
-
-# Definitions for monitoring the local (Linux) host
-cfg_file=@sysconfdir@/objects/localhost.cfg
-
-# Definitions for monitoring a Windows machine
-#cfg_file=@sysconfdir@/objects/windows.cfg
-
-# Definitions for monitoring a router/switch
-#cfg_file=@sysconfdir@/objects/switch.cfg
-
-# Definitions for monitoring a network printer
-#cfg_file=@sysconfdir@/objects/printer.cfg
+# Debian uses by default a configuration directory where icinga-common,
+# other packages and the local admin can dump or link configuration
+# files into.
+cfg_dir=/etc/icinga/objects/
 
 # Definitions for ido2db process checks
 #cfg_file=@sysconfdir@/objects/ido2db_check_proc.cfg
@@ -120,7 +97,7 @@ resource_file=@sysconfdir@/resource.cfg
 # for data retrieval (e.g. IDOUtils DB)
 # Note: This is a mandatory output for Icinga Classic UI to work properly.
 
-status_file=@STATEDIR@/status.dat
+status_file=/var/lib/icinga/status.dat
 
 
 
@@ -159,7 +136,7 @@ icinga_group=@icinga_grp@
 # you will have to enable this.
 # Values: 0 = disable commands, 1 = enable commands
 
-check_external_commands=1
+check_external_commands=0
 
 
 
@@ -188,8 +165,10 @@ command_check_interval=-1
 # is running as (usually 'nobody').  Permissions should be set at the 
 # directory level instead of on the file, as the file is deleted every
 # time its contents are processed.
+# Debian Users: In case you didn't read README.Debian yet, _NOW_ is the
+# time to do it.
 
-command_file=@EXTCMDFILEDIR@/icinga.cmd
+command_file=@localstatedir@/rw/icinga.cmd
 
 
 
@@ -247,7 +226,10 @@ event_broker_options=-1
 # ----> http://docs.icinga.org/latest/en/objectdefinitions.html
 #
 # Example definitions can be found in the '@sysconfdir@/modules/' directory.
-
+# If you want to enable idoutils in Debian install icinga-idoutils and copy
+# /usr/share/doc/icinga-idoutils/examples/idoutils.cfg-sample to
+# /etc/icinga/modules/idoutils.cfg.
+# Don't forget to also enable the daemon in /etc/default/icinga
 
 # LOG ROTATION METHOD
 # This is the log rotation method that Icinga should use to rotate
@@ -266,7 +248,7 @@ log_rotation_method=d
 # This is the directory where archived (rotated) log files should be 
 # placed (assuming you've chosen to do log rotation).
 
-log_archive_path=@LOGDIR@/archives
+log_archive_path=/var/log/icinga/archives
 
 
 
@@ -1250,7 +1232,7 @@ high_host_flap_threshold=20.0
 #	strict-iso8601	(YYYY-MM-DDTHH:MM:SS)
 #
 
-date_format=us
+date_format=iso8601
 
 
 
@@ -1280,7 +1262,7 @@ date_format=us
 # embedded Perl interpreter) is located.  If you didn't compile
 # Icinga with embedded Perl support, this option has no effect.
 
-p1_file=@P1FILELOC@/p1.pl
+p1_file=/usr/lib/icinga/p1.pl
 
 
 
@@ -1294,7 +1276,7 @@ p1_file=@P1FILELOC@/p1.pl
 # Only enable this setting when you really know what you are doing!
 # Values: 0 = disable interpreter, 1 = enable interpreter
 
-enable_embedded_perl=0
+enable_embedded_perl=1
 
 
 
@@ -1404,8 +1386,8 @@ use_true_regexp_matching=0
 # using the $ADMINEMAIL$ and $ADMINPAGER$ macros in your notification
 # commands.
 
-admin_email=@icinga_user@@localhost
-admin_pager=page@icinga_user@@localhost
+admin_email=root@localhost
+admin_pager=pageroot@localhost
 
 
 
@@ -1446,7 +1428,7 @@ use_large_installation_tweaks=0
 # Values: 1 - Enable environment variable macros
 #         0 - Disable environment variable macros (default)
 
-enable_environment_macros=0
+enable_environment_macros=1
 
 
 
@@ -1514,7 +1496,7 @@ debug_verbosity=2
 # DEBUG FILE
 # This option determines where Icinga should write debugging information.
 
-debug_file=@LOGDIR@/icinga.debug
+debug_file=/var/log/icinga/icinga.debug
 
 
 
