Description: Disable standard conforming strings.
Author: Alexander Wirt <formorer@debian.org>

--- a/module/idoutils/src/db.c
+++ b/module/idoutils/src/db.c
@@ -426,6 +426,9 @@
 #endif
 #ifdef USE_LIBDBI
 	const char *dbi_error;
+	const char *error_msg;
+	char *buf;
+	buf = NULL;
 #endif
 	ido2db_log_debug_info(IDO2DB_DEBUGL_PROCESSINFO, 2, "ido2db_db_connect() start\n");
 
@@ -529,6 +532,24 @@
 		idi->dbinfo.connected = IDO_TRUE;
 		syslog(LOG_USER | LOG_INFO, "Successfully connected to %s database", ido2db_db_settings.dbserver);
 	}
+
+
+	switch (idi->dbinfo.server_type) {
+		case IDO2DB_DBSERVER_PGSQL:
+			/* disable standardconforming strings for postgres,
+			 * this should make idoutils compatible with 9.1 until
+			 * there is a proper fix */
+			if (dbi_conn_query(idi->dbinfo.dbi_conn, "set standard_conforming_strings = off;") == NULL) {
+				dbi_conn_error(idi->dbinfo.dbi_conn, &error_msg);
+				syslog(LOG_USER | LOG_INFO, "Error: Could not unset standard_conforming_strings: %s\n", buf, error_msg);
+			} else {
+				syslog(LOG_USER | LOG_INFO, "Disable standard_conforming_strings in pgsql\n");
+			}
+			break;
+		default:
+			break;
+	}
+	free(buf);
 #endif
 
 #ifdef USE_PGSQL /* pgsql */
