Description: Fix DB upgrade.
Author: Alexander Wirt <formorer@debian.org>

--- a/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.3.0.sql
+++ b/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.3.0.sql
@@ -8,8 +8,6 @@
 -- Please check http://docs.icinga.org for upgrading information!
 -- -----------------------------------------
 
-CREATE LANGUAGE plpgsql;
-
 -- -----------------------------------------
 -- update dbversion
 -- -----------------------------------------
@@ -50,22 +48,6 @@ CREATE INDEX loge_inst_id_time_idx on ic
 
 
 -- -----------------------------------------
--- drop unique keys for check history
--- -----------------------------------------
-
-ALTER TABLE icinga_servicechecks DROP CONSTRAINT icinga_servicechecks_instance_id_key;
-
-ALTER TABLE icinga_hostchecks DROP CONSTRAINT icinga_hostchecks_instance_id_key;
-
--- -----------------------------------------
--- drop unique keys for * contacts
--- -----------------------------------------
-
-ALTER TABLE icinga_service_contacts DROP CONSTRAINT icinga_service_contacts_instance_id_key;
-ALTER TABLE icinga_host_contacts DROP CONSTRAINT icinga_host_contacts_instance_id_key;
-
-
--- -----------------------------------------
 -- add address6 column to hosts
 -- -----------------------------------------
 
--- a/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.5.0.sql
+++ b/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.5.0.sql
@@ -11,7 +11,6 @@
 -- drop unique keys for multiline inserts
 -- -----------------------------------------
 
-ALTER TABLE icinga_timeperiod_timeranges DROP CONSTRAINT uq_timeperiod_timeranges;
 ALTER TABLE icinga_host_parenthosts DROP CONSTRAINT uq_host_parenthosts;
 ALTER TABLE icinga_host_contactgroups DROP CONSTRAINT uq_host_contactgroups;
 ALTER TABLE icinga_service_contactgroups DROP CONSTRAINT uq_service_contactgroups;
